obelix/__init__.py

```diff
@@ -279,24 +298,120 @@ def read_data(self, data_path, no_cifs=False):
 class ShonAndMin(Dataset):
     '''
     Shon and Min dataset class.
-    
+
     Attributes:
-        dataframe (pd.DataFrame): DataFrame containing the dataset.
+        dataframe (pd.DataFrame): Cleaned DataFrame containing the dataset.
     '''
 
-    def __init__(self, data_path="/home/leah/leah---OBELiX/data/Shon_and_Min.xlsx"):
+    def __init__(self, data_path="./SM_rawdata"):
```

Comment:
I would add an argument to choose to clean the data (e.g. clean_data = True) and set it to true by default so that if someone wants the raw dataset they can have it.